### DISCUSSION TITLE
Where is the documentation of the extensions settings?

### DISCUSSION THREAD

**Discussion by pklapperich** [filips123/PWAsForFirefox#334] [Q&A] [answered]
On the Wiki there's a page called [Extension-Features](https://github.com/filips123/PWAsForFirefox/wiki/Extension-Features) that just says "Look at the README.md". 

The readme has a lot of _project level_ information. But where are all of the actual settings documented?
![image](https://github.com/filips123/PWAsForFirefox/assets/12666157/9d3dbb1e-5c24-47a6-9098-ecdbaa10593e)

Some are pretty straight forward. And "Enable automatic web app launching" self-documents (when you enable it the first time it opens a message that _tells you what it does_ which is awesome. But some of these settings aren't so obvious.

For example: I keep seeing "Always patch runtime and profile" suggested in questions and issues "Turn on this setting and try again, did that fix it?" But I have yet to see _an explanation of WHAT that setting even does_.

Likewise there's a lot of options in both Edit App and Edit Profile that I'm not entirely sure what they do. Is this documented somewhere? If it doesn't exist, then IMHO, this should just go on the existing "Extension-Features" page and the README.md should link out to the wiki.

**Comment by pklapperich** [comment]
I guess just "mouse over text" within the extension would go a long ways. If I could hover the mouse over "always patch" and got a 1-2 sentence explanation that would be super.

**Comment by filips123** [comment, marked as answer]
I know that documentation is lacking. I've been preparing new documentation as a website (built with `mkdocs-material` specifically) that will be better and more detailed for quite some time now. I didn't have much time lately and probably still won't have it for around two months, but then it should hopefully finally be ready.

I've now decided I'll soon just push what I have done so far so you and other users can suggest what you think should be improved. But it still won't be deployed as a website until I think it's ready for that.

**Reply by filips123** [reply]
I've pushed the prototype to the `wip-documentation` branch and created an announcment #335.
